automated/linux/piTest/test-PT-1.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+HW_UPDATE="The\sfirmware\sof\ssome\sI/O\smodules\smust\sbe\supdated."
+HW_NOT_PRESENT="Module\sis\sNOT\spresent"
+
+piTest -x
+if [ $? -eq 0 ]
+then
+    piTest -d | grep $HW_NOT_PRESENT
+    RET=$?
+
+    if [ $RET -ne 0 ]
+    then
+        lava-test-case logfile --result pass
+    else
+        lava-test-case logfile --result fail
+        lava-test-raise "Test PT-1 FAIL: DUT constellation must be checked! At least one module is NOT present, data is NOT available"
+    fi
+
+    #Check Module-UPDATE
+    piTest -d | grep $HW_UPDATE
+    if [ $? -eq 0 ]
+    then
+        echo "=====> The firmware of some I/O modules must be updated."
+        lava-test-case logfile --result fail
+    fi
+else
+    lava-test-case logfile --result fail
+    lava-test-raise "Test PT-1 FAIL: piTest -x ERROR"
```

Comment:
make the error message contain the check condition value ("$?") is always useful to debug the problem